src/doubleTrouble.js

```diff
@@ -27,6 +27,11 @@
  */
 function doubleTrouble(array, target) {
   // write code here
-}
-
+  for (var i = 0; i < array.length; i++) {
```

Comment:
[no var prefer let and const](https://mate-academy.github.io/style-guides/javascript.html#variables--const)